### PR TITLE
Add bsc

### PR DESCRIPTION
**Pull request by Darruma**
Add bsc setts to badger boost

### CODE REVIEW

File: rewards/boost/boost_utils.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
+from toolz.itertoolz import cons
```
Comment (mitche50) [inline rewards/boost/boost_utils.py:1]: what does this do? Don't see it referenced in the code

Comment (Darruma) [reply rewards/boost/boost_utils.py:1]: it was left there, feel free to remove

File: rewards/snapshot/chain_snapshot.py [hunk 2]
```diff
@@ -9,6 +9,8 @@
 
 console = Console()
 
+skip = ["0xb6bd5ae3d5f78a6bb04bbb031e24fa9c2bbd090d"]
```
Comment (mitche50) [inline rewards/snapshot/chain_snapshot.py:12]: Let's rename this to something that has more meaning than `skip`.  `invalid_vaults` maybe?

File: rewards/snapshot/chain_snapshot.py [hunk 3]
```diff
@@ -20,24 +22,26 @@ def chain_snapshot(chain: str, block: int):
...
-    chainBalances = fetch_chain_balances(chain, block - 50)
+    chainBalances = fetch_chain_balances(chain, block)
     balancesBySett = {}
 
     for settAddr, balances in list(chainBalances.items()):
-        settBalances = parse_sett_balances(settAddr, balances, chain)
-        token = interface.IERC20(settAddr)
-        console.log("Fetched {} balances for sett {}".format(len(balances), token.name()))
+        settBalances = parse_sett_balances(settAddr, balances)
+        # token = interface.IERC20(settAddr)
```
Comment (mitche50) [inline rewards/snapshot/chain_snapshot.py:30]: let's not leave hanging comments for code cleanliness

**Pull request closed without merging**